README.md

```diff
@@ -52,7 +52,7 @@ You will need the following:
 
 * JVM 8
 * Android SDK and the latest stable Android Studio or IntelliJ
-* Intellij Kotlin plugin with 1.3.71 support
+* Intellij Kotlin plugin with 1.3.71 support (should be included in the latest Android Studio)
 * Mac with Xcode 11+ for the iOS build
 
 ### Important Note (Feb 11)
```

Comment:
Wondering if we can drop this too. Came up originally because 3.5 had some issues that were fixed in 3.6.

Reply:
Put version req's in the list above. even that might be more than needed though